ThingIFSDK/ThingIFSDK/ThingIFAPI+Push.swift

```diff
@@ -69,29 +69,31 @@ extension ThingIFAPI {
        installationID property is used.
      */
     open func uninstallPush(
-        _ installationID:String?,
+        _ installationID: String? = nil,
         completionHandler: @escaping (ThingIFError?)-> Void
         )
     {
-        let idParam = installationID != nil ? installationID : self.installationID
-        let requestURL = "\(baseURL)/api/apps/\(appID)/installations/\(idParam!)"
+        guard let installationID = installationID ?? self.installationID else {
+            completionHandler(ThingIFError.unsupportedError)
```

Comment:
@yongpingchen In thing-if iOSSDK, we use `thingIFErrro.unsupportedError` for invalid parameter. I followed other implementations.

One example is [here](https://github.com/KiiPlatform/thing-if-iOSSDK/blob/impl-trait-api/ThingIFSDK/ThingIFSDK/ThingIFAPI+OnBoard.swift#L118).